.travis.yml

```diff
@@ -1,35 +1,62 @@
 language: php
+sudo: false
+cache:
+    directories:
+        - $HOME/.composer/cache/files
+        - $HOME/symfony-bridge/.phpunit
 
-php:
-    - 5.5
-    - 5.6
-    - 7.0
-    - 7.1
-    - 7.2
-    - nightly
+env:
+    global:
+        - PHPUNIT_FLAGS="-v"
```

Comment:
As we always want the `-v` part, it could stay directly in the command, removing this variable here.